vulnfeeds/cpp/main.go

```diff
@@ -131,6 +131,41 @@ func GitVersionsToCommit(CVE string, versions cves.VersionInfo, repos []string,
 	return v, nil
 }
 
+func refAcceptable(ref cves.CVEReferenceData, tagDenyList []string) (acceptable bool) {
```

Comment:
nit:  the return variable `acceptable` seems unused

Reply:
Fixed in [b2298f2](https://github.com/google/osv.dev/pull/1164/commits/b2298f296852fd5a931bb67374faaa0c733701fa)